pennylane/operation.py

```diff
@@ -241,7 +241,7 @@ def _matrix(cls, *params):
         """Matrix representation of the operator
         in the computational basis.
 
-        This is a *static method* that should be defined for all
+        This is a *class method* that should be defined for all
```

Comment:
Don't forget to also update https://pennylane.readthedocs.io/en/latest/development/plugins.html#supporting-new-operations